config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -44,6 +44,44 @@ presubmits:
           requests:
             cpu: 7
             memory: 12Gi
+  - name: pull-kubernetes-verify-strict-lint
+    cluster: k8s-infra-prow-build
+    always_run: true
```

Comment:
please start with always_run: false which is a cheaper way to experiment. (you can manually invoke it with `/test` on a test PR)

Reply:
Changed.